diagnostic.md

```diff
@@ -150,7 +171,8 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+divTwo, divThree, divFour, and divFive will appear next to each other, respectively, to the left
```

Comment:
```.divOne``` will be in the top-right corner;
```.divTwo``` will be in the top-left corner, with .divThree immediately to its right;
```.divFour``` will be underneath ```.divTwo```, and ```.divFive``` will sit immediately to the right of ```.divFour``` (under ```.divThree```).